.github/workflows/test.yml

```diff
@@ -4,6 +4,10 @@ name: CI
 
 on: [push, pull_request]
 
+defaults:
+  run:
+    shell: bash
```

Comment:
Run commands with pipefail.
See <https://github.com/lowlighter/libs/pull/69#issuecomment-2212241938:~:text=It%20runs%20with,faster)%20(ok%20reason)> for a longer explaination.